orientdb/src/main/java/com/yahoo/ycsb/db/OrientDBClient.java

```diff
@@ -46,6 +49,7 @@
   private static final String             CLASS = "usertable";
   protected ODatabaseDocumentTx             db;
   private ODictionary<ORecord> dictionary;
+  private boolean isRemote = false;
```

Comment:
Whether or not the connection is a remote connection is purely dependent on the url provided by the user, so really there is not default. ```isRemote``` is defaulted to false because it is only set to true if the user provides a remote connection url, in any other case (user provides a local url) it should be false.